webserver/webserver/src/main/java/io/helidon/webserver/LoomServer.java

```diff
@@ -53,6 +53,7 @@ class LoomServer implements WebServer {
     private static final AtomicInteger WEBSERVER_COUNTER = new AtomicInteger(1);
 
     private final Map<String, ServerListener> listeners;
+    private final Runnable afterStartNotifier;
```

Comment:
Can this be a private method instead of a `Runnable` ?

Reply:
OK